### ISSUE TITLE
Vision Titans

### ISSUE THREAD

**Issue by ShunkaiZhou** [lucidrains/titans-pytorch#7]
Thank you very much for the excellent code!
May I ask if you are considering building Vision Titans?
Best wishes

**Comment by lucidrains** [comment]
@ShunkaiZhou hey Shunkai, thanks for your interest

could you elaborate a bit more? do you mean in the context of tokenized video or something different?

**Comment by ShunkaiZhou** [comment]
@lucidrains Hi, thank you for your quick reply.
For example, online video analytics?
Best wishes.

**Comment by kanghuazhao** [comment]
VLM-Titans？

**Comment by RefractAI** [comment]
@lucidrains 

I would like to share that, by replacing the logits with a continuous pixel prediction, this repo generates class-conditional MNIST digits images in under 2 minutes on a 4090. Hyperparameters unchanged from default.

![Image](https://github.com/user-attachments/assets/115e3464-96d8-4a90-99d4-fe4559a3e955)

**Comment by lucidrains** [comment]
@RefractAI nice!

**Comment by ShunkaiZhou** [comment]
@RefractAI 
Hi, your changes are very interesting. Could you share the changed code please? Thanks a lot.
Best wishes!

**Comment by samb271** [comment]
Hi! Is there an existing implementation of memory on an encoder-only ViT?
